<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1290</width>
      <height>900</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.358, y1:0.244, x2:1, y2:1, stop:0.353234 rgba(39, 55, 77, 255), stop:1 rgba(82, 109, 130, 255))
</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>300</y>
       <width>450</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(0, 0, 0, 70);
	border-radius: 10px;
}

QWidget:hover{
	background-color: rgba(71, 78, 104, 50);

}</string>
     </property>
     <widget class="QScrollArea" name="scrollAreaDetails">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>631</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(157, 178, 191);
background-color: rgba(0, 0, 0, 0);
border-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>450</width>
         <height>631</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>450</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(0, 0, 0, 70);
	border-radius: 10px;
}

QWidget:hover{
	background-color: rgba(71, 78, 104, 50);

}</string>
     </property>
     <widget class="QScrollArea" name="scrollAreaDetails_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>631</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(157, 178, 191);
background-color: rgba(0, 0, 0, 0);
border-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>450</width>
         <height>631</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1290</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: qlineargradient(spread:pad, x1:0.776, y1:0.614091, x2:0, y2:0.999682, stop:0.19403 rgba(82, 109, 130, 255), stop:1 rgba(31, 110, 140, 255))</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1197</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 35px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logo app.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="goHomeBtn">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(157, 178, 191);
	border-radius: 35px;
	border : 2px solid black;
	border-color: rgb(46, 138, 153);
}

QPushButton:hover{
	background-color: rgb(132, 167, 161);
	border : 3px solid black;
	border-color: rgb(100, 204, 197)
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>home.png</normaloff>home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="profileInfoBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(157, 178, 191);
	border-radius: 35px;
	border : 2px solid black;
	border-color: rgb(46, 138, 153);
}

QPushButton:hover{
	background-color: rgb(132, 167, 161);
	border : 3px solid black;
	border-color: rgb(100, 204, 197)
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>profile.png</normaloff>profile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>100</y>
       <width>450</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(70, 69, 140, 255), stop:1 rgba(27, 107, 147, 255));
border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="photoProduct1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 189, 189);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nameProduct1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 10);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="priceProduct1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 10);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_11" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgba(0, 0, 0, 0);;
}

QWidget:hover{
	background-color: rgba(0, 0, 0, 50);;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>450</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(70, 69, 140, 255), stop:1 rgba(27, 107, 147, 255));
border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="photoProduct2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 189, 189);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nameProduct2">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>30</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 10);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="priceProduct2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 10);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_10" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgba(0, 0, 0, 0);;
}

QWidget:hover{
	background-color: rgba(0, 0, 0, 50);;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="clearCompareList">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>delete.png</normaloff>delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>63</width>
       <height>46</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
